pkg/scraper/image.go

```diff
@@ -0,0 +1,75 @@
+package scraper
+
+import (
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/stashapp/stash/pkg/models"
+	"github.com/stashapp/stash/pkg/utils"
+)
+
+func setPerformerImage(p *models.ScrapedPerformer) error {
+	if p == nil || p.Image == nil || !strings.HasPrefix(*p.Image, "http") {
+		// nothing to do
+		return nil
+	}
+
+	img, err := getImage(*p.Image)
+	if err != nil {
+		return err
+	}
+
+	p.Image = img
+
+	return nil
+}
+
+func setSceneImage(s *models.ScrapedScene) error {
+	// don't try to get the image if it doesn't appear to be a URL
+	if s == nil || s.Image == nil || !strings.HasPrefix(*s.Image, "http") {
+		// nothing to do
+		return nil
+	}
+
+	img, err := getImage(*s.Image)
+	if err != nil {
+		return err
+	}
+
+	s.Image = img
+
+	return nil
+}
+
+func getImage(url string) (*string, error) {
+	// assume is a URL for now
+	resp, err := http.Get(url)
```

Comment:
should we use a new http client with a default appropriate timeout instead of a plain get ?